1-org/modules/centralized-logging/main.tf

```diff
@@ -124,6 +151,24 @@ resource "google_project_iam_member" "logbucket_sink_member" {
   member = module.log_export["${each.value}_lbk"].writer_identity
 }
 
+#------------------------------------------------------------------#
+# Log Bucket Service account IAM membership for log_export_billing #
+#------------------------------------------------------------------#
+resource "google_project_iam_member" "logbucket_sink_member_billing" {
+  count = var.enable_billing_account_sink == true && var.logbucket_options != null ? 1 : 0
+
+  project = var.logging_destination_project_id
+  role    = "roles/logging.bucketWriter"
+
+  # Set permission only on sinks for this destination using
+  # module.log_export_billing key "<resource>_<dest>"
+  member = module.log_export_billing["_lbk"].writer_identity
```

Comment:
is the value `"_lbk"` correct?

Reply:
and it should be called `destination_resource_name` not `destination_resource_uri`